mypy/plugin.py

```diff
@@ -340,15 +341,15 @@ def class_type(self, self_type: Type) -> Type:
         raise NotImplementedError
 
     @abstractmethod
-    def lookup_fully_qualified(self, name: str) -> SymbolTableNode:
+    def lookup_fully_qualified(self, fullname: str, /) -> SymbolTableNode:
```

Comment:
```suggestion
    def lookup_fully_qualified(self, fullname: str) -> SymbolTableNode:
```
I'd again update the implementation instead.
https://github.com/python/mypy/blob/ac6151ae6657a8bbd9793c74691c9985dae02c15/mypy/typeanal.py#L291-L292